draft-ietf-jsonpath-base.md

```diff
@@ -1257,13 +1255,16 @@ Children of an array appear in array order in the resultant nodelist.
 {: unnumbered}
 
 A path by itself in a Logical context is an existence test which yields true if the path selects at least one node and yields false if the path does not select any nodes.
+A function expression with result type `SingleNodeType` or `NodesType` in a Logical context is an existence test
+which yields true if the function returns a non-empty nodelist and yields false if the function returns an empty node-list.
 
 Existence tests differ from comparisons in that:
 
 * they work with arbitrary relative or absolute paths (not just Singular Paths).
 * they work with paths that select structured values.
+* they work with function expressions with result type `SingleNodeType` or `NodesType` (not just `SingleNodeType`).
 
-To test the value of a node selected by a path, an explicit comparison is necessary.
+To test the value of a node selected by a path or returned by a function expression, an explicit comparison is necessary.
```

Comment:
I think we're changing context from "test expression" to "comparison" here, but that context shift isn't clear.

Reply:
The word "comparison" makes me think it is a comparison context.

How does sentence this apply to a test expression?  It's talking about testing values, which isn't a thing we do.  To be honest, this whole sentence is confusing, and I think it can probably just be removed.

Reply:
s/test/examine/

Reply:
I've added that word substitution to PR #420. Please can we continue the rewording there as it's more relevant to the base than the subject of the current PR?